From 23911114d030449f0de4cb6f8690b5e096018f89 Mon Sep 17 00:00:00 2001
From: Chen Zhongjin <chenzhongjin@huawei.com>
Date: Mon, 20 Feb 2023 03:45:19 -0500
Subject: [PATCH 059/500] [Huawei] bpf: Make BPF_XDP_UPDATE_TCP_SEQ_HELPER
 default to disabled

Offering: HULK
hulk inclusion
category: feature
bugzilla: N/A

--------------------------------

It is not necessary to enable BPF_XDP_UPDATE_TCP_SEQ_HELPER defaultly.
Set it to n in net/Kconfig.

Fixes: a71b79f66b1d ("[Huawei] samples: bpf: Minior fixes for BMC sample")
Signed-off-by: Chen Zhongjin <chenzhongjin@huawei.com>
---
 net/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/Kconfig b/net/Kconfig
index 1ddfc8693a0f..db3cb5e52cd8 100644
--- a/net/Kconfig
+++ b/net/Kconfig
@@ -332,7 +332,7 @@ config EULER_SOCKETMAP
 
 config BPF_XDP_UPDATE_TCP_SEQ_HELPER
 	bool "enable XDP helper bpf_update_tcp_seq"
-	default y
+	default n
 	help
 	  Enable ebpf helper to allows XDP synchronize the TCP socket status.
 	  This helper will update socket status directly only with minimal
-- 
2.17.1

